File: algorithm/python/ExcelSheetColumnNumber.py
class Solution:
    def titleToNumber(self, columnTitle):
        length = len(columnTitle)
        res = 0
        for i in range(0, length):
            num = ord(columnTitle[i]) - ord('A') + 1
            idx = length - i -1
            res += num * pow(26, idx)
        
        return res

File: algorithm/JavaScript/ExcelSheetColumnNumber.js
/**
 * @param {string} columnTitle
 * @return {number}
 */
 var titleToNumber = function(columnTitle) {
    let res =0
    for (let i = 0; i < columnTitle.length; i++) {
        const num = columnTitle[i].charCodeAt() - 'A'.charCodeAt() + 1
        const idx = columnTitle.length - i - 1
        res += num * (26 ** idx)
    }
    return res
};